src/components/register/register.css

```
@@ -572,6 +577,26 @@ body {
   -webkit-box-orient: vertical;
 }
+.book-row-synopsis.expanded {
+  display: block;
+  -webkit-line-clamp: unset;
+}
+.synopsis-toggle {
+  background: none;
+  border: none;
+  padding: 0;
+  margin-top: 2px;
+  font-family: "Inter", sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)